modules/salt/pages/salt-states.adoc

```diff
@@ -70,6 +70,17 @@ Apply the highstate to all clients in the group:
 salt -I 'group_ids:GID' state.apply
 ----
 
+To apply a custom state at highstate create a mapping in [path]``/srv/salt/top.sls``.
+This short example maps the [literal]``test`` state to the system group [literal]``12``:
+
+----
+# /srv/salt/top.sls
+base:
+  'group_ids:12':
+    - match: pillar
+    - test
+----
+
```

Comment:
So this probably needs to go in the new Custom Salt States section now.

Reply:
I moved it. Please, recheck!